src/library_fs.js

```diff
@@ -1361,24 +1361,35 @@ FS.staticInit();` +
       FS.mkdir('/proc/self/fd');
       FS.mount({
         mount: () => {
-          var node = FS.createNode(proc_self, 'fd', {{{ cDefine('S_IFDIR') }}} | 511 /* 0777 */, {{{ cDefine('S_IXUGO') }}});
+          var node = MEMFS.createNode(proc_self, 'fd', {{{ cDefine('S_IFDIR') }}} | 511 /* 0777 */, {{{ cDefine('S_IXUGO') }}});
           node.node_ops = {
+            getattr: MEMFS.node_ops.getattr,
```

Comment:
I wonder if we should instead just override the `lookup` and `readdir` attributes.

e.g.

```
node.node_ops = Object.assign(node.node_ops, {
   ...
}

Reply:
That way we would get all the default node_ops from MEMFS

Reply:
IMO this would break compability whith *NIX systems, since the procfs filesystem also dosn't allow any writing/deleting operation.